Plugins/GitUIPluginInterfaces/IGitModule.cs

```diff
@@ -156,8 +156,31 @@ public interface IGitModule
         Task<IReadOnlyList<Remote>> GetRemotesAsync();
 
         string GetSetting(string setting);
+
+#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved
```

Comment:
Move `GitConfigFormatException` to `Plugins/GitUIPluginInterfaces`?

Reply:
Good call - we can't have the upstream dependency refer to downstream implementations.